plugin/storage/integration/es_integration_test.go

```diff
@@ -75,6 +75,9 @@ func (s *ESStorageIntegration) initializeES() error {
 
 func (s *ESStorageIntegration) esCleanUp() error {
 	_, err := s.client.DeleteIndex("*").Do(context.Background())
+	client := es.WrapESClient(s.client)
```

Comment:
nit: dry this up (the same code exists on L61)

Also maybe we should consider having a TTL of 0 means no cache